go.mod

```diff
@@ -27,7 +27,7 @@ require (
 	github.com/multiformats/go-multiaddr v0.14.0
 	github.com/multiformats/go-multibase v0.2.0
 	github.com/multiformats/go-multihash v0.2.3
-	github.com/storacha/go-libstoracha v0.0.1
+	github.com/storacha/go-libstoracha v0.0.2-0.20250408211255-f4eb984a91fe
```

Comment:
https://github.com/storacha/go-libstoracha/pull/14 needs to land before this, then we I need to update this ref before merge.